src/components/Prescription/Prescription.css

```diff
@@ -54,6 +54,6 @@
 }
 
 .script-show-name {
-    font-size: 400%;
+    font-size: 300%;
```

Comment:
also changed this for the above reason, the text was a bit too large and pushing the link below the 'top-container' border